paraver build errors out when install the "wxpropgrid" dependency

While installing paraver, as a dependency of it, I get an error installing wxpropgrid in `spack/var/spack/repos/builtin/packages/wxpropgrid/package.py:23` as,

```python
         21    def install(self, spec, prefix):
         22        configure("--prefix=%s" % prefix, "--with-wxdir=%s" %
  >>     23                  spec['wx'].prefix.bin, "--enable-unicode")
         24
         25        make()
         26        make("install")
```

This seems to be a bug, and I made the following modification

```diff
--- a/var/spack/repos/builtin/packages/wxpropgrid/package.py
+++ b/var/spack/repos/builtin/packages/wxpropgrid/package.py
@@ -20,7 +20,7 @@ class Wxpropgrid(Package):
 
     def install(self, spec, prefix):
         configure("--prefix=%s" % prefix, "--with-wxdir=%s" %
-                  spec['wx'].prefix.bin, "--enable-unicode")
+                  spec['wxwidgets'].prefix.bin, "--enable-unicode")
 
         make()
         make("install")
```

With this change in place, the installation process of `wxpropgrid` manages to reach the configuration step. But, I get another error. Here's the build-log:

```
==> Executing phase: 'install'                                                                                                                                                           
==> [2019-11-08-09:09:16.185348] './configure' '--prefix=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxpropgrid-1.4.15-4pyki3ldvmeelfu52ntq22baqy2nte5o' '--with-wxdir=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxwidgets-3.1.0-zartvb5bcg5mddj4n7smzindfkcbg7mq/bin' '--enable-unicode'
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
checking for the --enable-debug option... will be automatically detected
checking for the --enable-unicode option... yes 
checking for the --enable-shared option... will be automatically detected
checking for the --with-toolkit option... will be automatically detected
checking for the --with-wxshared option... will be automatically detected
checking for the --with-wxversion option... will be automatically detected
checking for gcc... /home/vsoni/tools/spack/lib/spack/env/gcc/gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes 
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes 
checking whether /home/vsoni/tools/spack/lib/spack/env/gcc/gcc accepts -g... yes 
checking for /home/vsoni/tools/spack/lib/spack/env/gcc/gcc option to accept ANSI C... none needed
checking whether we are using the GNU C++ compiler... yes 
checking whether /home/vsoni/tools/spack/lib/spack/env/gcc/g++ accepts -g... yes 
checking for wx-config... /home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxwidgets-3.1.0-zartvb5bcg5mddj4n7smzindfkcbg7mq/bin/wx-config
checking for wxWidgets version >= 2.6.1 (--unicode=yes)... yes (version 3.1.0)
checking for wxWidgets static library... no
checking if wxWidgets was built in DEBUG mode... no
checking if wxWidgets was built in STATIC mode... no
checking which wxWidgets toolkit was selected... configure: error:
        Cannot detect the currently installed wxWidgets port !
        Please check your 'wx-config --cxxflags'...
```

And, here's the build-env

<details>
  <summary>build-env</summary>

```shell
ACLOCAL_PATH=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pkgconf-1.6.3-bpu62yhug3cotbexuhsdhn6573cuahwr/share/aclocal:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/util-macros-1.19.1-lzac7qiqtqjixxnje3ithk3eumyew727/share/aclocal:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libtool-2.4.6-puue3pm7jv6vzdyg2b7x3sbwm77whkp6/share/aclocal; export ACLOCAL_PATH
BASH_ENV=/usr/share/modules/init/bash; export BASH_ENV
CADNA_ROOT=/home/vsoni/tools/cadna-3.1.3; export CADNA_ROOT
CC=/home/vsoni/tools/spack/lib/spack/env/gcc/gcc; export CC
CLUTTER_IM_MODULE=xim; export CLUTTER_IM_MODULE
CMAKE_PREFIX_PATH=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/swig-4.0.0-yqnn4c7zaxgc4smdgeqqzuryndx7acgk:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/bzip2-1.0.8-bl277ctefakb6ff4kxdxkxxpgtc7zpdk:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libjpeg-turbo-2.0.3-qu75rijvkj3bvptcyx3v5wmyqjm6a72u:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/ncurses-6.1-giqjoryaltcr2gj227fdhapqzozqbxue:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/tar-1.32-we74er7tv5hkuirva2tvlq35xjagrzbb:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/glib-2.56.3-ncbtvgcc33233flrixxmuyhex54nqafu:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxfixes-5.0.2-ld3isqoniw52emmjc4dgej3fsfrkiv4n:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-atk-2.26.2-wwqrhwzbcelwyfwcsifykyrbufgj2t4g:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libbsd-0.9.1-nzolvhj75tmbzgtsycmnmispon2a5lne:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/python-3.7.4-k4keiszfsytvqkmrn44g3dbglkn3n7iq:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libedit-3.1-20170329-ceqgzwvgiar3mswya7iv3ayjokncfkzi:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/icu4c-64.1-lmyzw76ojwniyhgq3ztwmgniihnbjtfd:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gobject-introspection-1.56.1-dxqvlfxwwkvvlxex2nzgnl3u6u6p42zb:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libtiff-4.0.10-56m4wnmsgy6speokfawaw7ezn3pjnpha:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxi-1.7.6-zhwscljazuzdxd7d4h4brynj22wwbpx6:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gtkplus-3.20.10-z7424szywk3cog6dzgfhxo5xdwug3jno:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/sqlite-3.30.1-sbgek46asjt6gx475p3o264aoalkiiez:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/inputproto-2.3.2-nundvjesicbpkcrosjvwhggrzbt2gh3j:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/binutils-2.32-ehnjwaojurg4n6vbbgsdfp7aho7qwvvo:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/freetype-2.10.1-3nyzzr7hrfmrtvjziucbhf7ls2m5mocq:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxft-2.3.2-wozy34ikvfbplhmk6mfwcapt5jtllt75:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/font-util-1.3.2-wmlz4nwnnj7jhxbrg6age6ziw5nwrs5v:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libepoxy-1.4.3-3jhwzitxd6kem47je74xtojbzw2egvcb:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdbm-1.18.1-o3gn2zefwtl2qopdw3hpznlzxbm6ufvd:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdk-pixbuf-2.38.0-fer36fxn3xqf53yrohvbh6vxqylvkjfg:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/harfbuzz-2.3.1-3f2tn3mmfa2c5w5z2syhengrhv3qbh7f:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/shared-mime-info-1.9-4c7mld6ugi4fxaqiwz5e6o747vw7e25v:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxml2-2.9.9-rsomm2rw7otzkjbqja46yfr3i4dk32bk:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/kbproto-1.0.7-bbyqmd4rg7nd2bck6tur6lahnw6hvojd:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxrender-0.9.10-cdsuuaxbchxcgupekrozvxs3h2ncgnhz:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-core-2.28.0-74hsipz5ctek25xcecrjk5puevmli7wl:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/openssl-1.1.1d-q2wzstunrtm256ctqhqzplruwi7msedn:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpng-1.6.37-znjevoldm5l2cmwr5w6mzwqcp6mdiieq:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xproto-7.0.31-caalnyvvme5wfiac6mkraqu2jggxug4o:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pixman-0.38.0-altkkt5wlktit32npwgogpi7qlfap4qi:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/mesa-18.3.6-m2njhpfhxpxn3nth5sws5z7ys65dzvbh:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/expat-2.2.9-mnfcw7ow22ukhji4ypq43ty4qu454f33:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxext-1.3.3-as6foiyy32m47pzhscqpq6alvs373c33:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libffi-3.2.1-5wasckgclanmn27zgih4gqfdg72gnsts:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxwidgets-3.1.0-zartvb5bcg5mddj4n7smzindfkcbg7mq:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xz-5.2.4-fczgkodctns3rrkjxydtw7utm3hgtszb:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/dbus-1.12.8-qz3lq5alzkjcu7bo2txav35upfarsg7w:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/readline-8.0-7s3tknmsgd2ogp5i2fxanncjwwlz5imd:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fixesproto-5.0-s2y5coudhip4s5tmbtnm45izlqx5v6li:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpthread-stubs-0.4-6y5jawqhzywitj5qau4zytfqrhe322nj:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pcre-8.42-grb7dd2yxp6dik2wpk3gcxxviccn66i6:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pango-1.41.0-7fm7ua5nhdamvfcwwpcdeafk35zdabqu:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/atk-2.30.0-donciffdv5sjdflnt2o3o3rac7fgkelc:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxau-1.0.8-j5wav4lxjwarzlydjjgxka3kbkqm6gyf:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libiconv-1.16-qhwzzrmbtm6myg523nyh6yb42q4tuam3:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/llvm-9.0.0-q5vpxksi4qdfaonkvqwtejh24wtf3eqw:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/zlib-1.2.11-6jyao6lrmcwyq7czbm6rkotf2kotdkms:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gettext-0.20.1-xno722wxyggfz6ako7u6dnc2vikmmv4e:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xextproto-7.3.0-mu4d2pxsnfovkjf5xiz2jifvmr7mvhfl:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxcb-1.13-4it4r7ddtsqomtfb2b73i64ekghovboo:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fontconfig-2.12.3-zct5jz7bzilfao2tbhm56rxt2ezrw4of:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libx11-1.6.7-xiug4qf74mreb4awmuiyhdx3jxwxlc3x:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/meson-0.49.1-5u5phgd74iiao54lfeh5urqzslrmkwuq:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxdmcp-1.1.2-6izlm3rd6hazdewvnultwwbhakjdksbm:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/renderproto-0.11.1-emcx3vuhvblg44pvs3c4sor5ihdfyc5w:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/cairo-1.16.0-2jluafsrkf5neehysrkso5tau5ihxcf6; export CMAKE_PREFIX_PATH
COLORTERM=truecolor; export COLORTERM
CXX=/home/vsoni/tools/spack/lib/spack/env/gcc/g++; export CXX
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus; export DBUS_SESSION_BUS_ADDRESS
DESKTOP_AUTOSTART_ID=10a9fd46859cec1906157319982761966000000032410007; export DESKTOP_AUTOSTART_ID
DISPLAY=:10.0; export DISPLAY
DOCKER_HOME=/home/dev/; export DOCKER_HOME
ENV=/usr/share/modules/init/profile.sh; export ENV
EXA2PRO_SANDBOX=/home/vsoni/exa2pro-sandbox; export EXA2PRO_SANDBOX
F77=/home/vsoni/tools/spack/lib/spack/env/gcc/gfortran; export F77
FC=/home/vsoni/tools/spack/lib/spack/env/gcc/gfortran; export FC
GNOME_DESKTOP_SESSION_ID=this-is-deprecated; export GNOME_DESKTOP_SESSION_ID
GNOME_SHELL_SESSION_MODE=ubuntu; export GNOME_SHELL_SESSION_MODE
GNOME_TERMINAL_SCREEN=/org/gnome/Terminal/screen/639267c7_9a7a_4181_8184_d64eda730537; export GNOME_TERMINAL_SCREEN
GNOME_TERMINAL_SERVICE=:1.76; export GNOME_TERMINAL_SERVICE
GPG_AGENT_INFO=/run/user/1000/gnupg/S.gpg-agent:0:1; export GPG_AGENT_INFO
GTK_IM_MODULE=ibus; export GTK_IM_MODULE
GTK_MODULES=gail:atk-bridge; export GTK_MODULES
HOME=/home/vsoni; export HOME
IM_CONFIG_PHASE=2; export IM_CONFIG_PHASE
INTELOPENCLSDK=/home/vsoni/tools/opencl_sdk/system_studio_2019/opencl/SDK; export INTELOPENCLSDK
LANG=en_US.UTF-8; export LANG
LC_ADDRESS=fr_FR.UTF-8; export LC_ADDRESS
LC_ALL=C; export LC_ALL
LC_IDENTIFICATION=fr_FR.UTF-8; export LC_IDENTIFICATION
LC_MEASUREMENT=fr_FR.UTF-8; export LC_MEASUREMENT
LC_MONETARY=fr_FR.UTF-8; export LC_MONETARY
LC_NAME=fr_FR.UTF-8; export LC_NAME
LC_NUMERIC=fr_FR.UTF-8; export LC_NUMERIC
LC_PAPER=fr_FR.UTF-8; export LC_PAPER
LC_TELEPHONE=fr_FR.UTF-8; export LC_TELEPHONE
LC_TIME=fr_FR.UTF-8; export LC_TIME
LESSCLOSE='/bin/lesspipe %s %s'; export LESSCLOSE
LESSOPEN='| /bin/lesspipe %s'; export LESSOPEN
LOADEDMODULES=''; export LOADEDMODULES
LOGNAME=vsoni; export LOGNAME
LS_COLORS='rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:'; export LS_COLORS
MODULEPATH=/home/vsoni/tools/spack/share/spack/modules/linux-ubuntu18.04-skylake:/etc/environment-modules/modules:/usr/share/modules/versions:/usr/share/modules//modulefiles:/usr/share/modules/modulefiles:/home/vsoni/tools/modulefiles; export MODULEPATH
MODULEPATH_modshare='/usr/share/modules/$MODULE_VERSION/modulefiles:1:/etc/environment-modules/modules:1:/home/vsoni/tools/modulefiles:1:/usr/share/modules/modulefiles:1:/usr/share/modules/versions:1'; export MODULEPATH_modshare
MODULESHOME=/usr/share/modules; export MODULESHOME
MODULES_CMD=/usr/lib/x86_64-linux-gnu/modulecmd.tcl; export MODULES_CMD
#OLDPWD=/home/vsoni; export OLDPWD
PATH=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/python-3.7.4-k4keiszfsytvqkmrn44g3dbglkn3n7iq/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-xml-parser-2.44-nzlivs6eyhbim5tcwvpwxdqr7dsdhqi3/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-libwww-perl-6.33-fu23akf3oubl6oexnjr75k5lnf6uueyb/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-encode-locale-1.05-uh4ixoroml55eark7m5bw3llob77xh42/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-file-listing-6.04-idbsotp6zezxrr4mocsrxf2mxabxsme2/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-date-6.02-vqus7asf43p2iqzarbtzuo5fsdx27sqt/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-html-parser-3.72-sltkayjoyqsahyqxng5o4fpsses42r7n/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-html-tagset-3.20-gmhueubhfwqprjlv6sgi27yj5egmfs7g/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-cookies-6.04-6ry3g43avudslc67nmarfuwdcfvco7uo/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-message-6.13-abohrnrxuisfhk3t72seiv574bc3v4mc/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-io-html-1.001-oyqd2erifynoeupo66hjcaycihsicoqq/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-lwp-mediatypes-6.02-bdp3xi7cv2fdcauoihwrard42kepoxia/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-try-tiny-0.28-zvtluoeno7jqe67p2qzwwubchfxa4kz4/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-uri-1.72-o37xz4ilf4rnmda2svvtotxm4xeam2ui/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-test-needs-0.002005-tnrswihcn4zxwfksomggqzarz33rnl7y/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-daemon-6.01-rko6s7gedbap2xxkdpus4zp74zt3dvfw/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-module-build-tiny-0.039-mmndrhwxlan4msx5ad2c5ohhwjpiu33m/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-extutils-config-0.008-ftjvuhon5zjjsgap5ortxwd7xbkshsph/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-extutils-helpers-0.026-stpvmn3lj5ue2qkblyys53jkuza6cl57/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-extutils-installpaths-0.012-uwbd4w3fdwy5mmsqfjvryiw2eyopbz2z/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-module-build-0.4224-t3cop5s3douirz5q55bvwbkhfadwd4el/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-negotiate-6.01-ommofkbrsj7u6ainlq5eeksb5nlcgymc/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-net-http-6.17-omo2m5srfju7nxpljyxzjashh2unpzvg/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-www-robotrules-6.02-uf5nlvfqshw7rbvtpnzcrmfaj5djiun2/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-data-dumper-2.173-hxgfazm472z25crph4yyvurdrwr37563/bin:/home/vsoni/tools/spack/lib/spack/env/gcc:/home/vsoni/tools/spack/lib/spack/env/case-insensitive:/home/vsoni/tools/spack/lib/spack/env:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxwidgets-3.1.0-zartvb5bcg5mddj4n7smzindfkcbg7mq/bin:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxwidgets-3.1.0-zartvb5bcg5mddj4n7smzindfkcbg7mq/bin:/home/vsoni/tools/spack/bin:/home/vsoni/tools/openmpi/bin:/home/vsoni/tools/julia-1.1.0/bin:/home/vsoni/tools/pdt/x86_64/bin:/home/vsoni/tools/tau/x86_64/bin:/home/vsoni/tools/likwid/bin:/sbin:/bin:/usr/bin:/usr/local/bin:/snap/bin; export PATH
PERL5LIB=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-xml-parser-2.44-nzlivs6eyhbim5tcwvpwxdqr7dsdhqi3/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-libwww-perl-6.33-fu23akf3oubl6oexnjr75k5lnf6uueyb/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-encode-locale-1.05-uh4ixoroml55eark7m5bw3llob77xh42/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-file-listing-6.04-idbsotp6zezxrr4mocsrxf2mxabxsme2/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-date-6.02-vqus7asf43p2iqzarbtzuo5fsdx27sqt/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-html-parser-3.72-sltkayjoyqsahyqxng5o4fpsses42r7n/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-html-tagset-3.20-gmhueubhfwqprjlv6sgi27yj5egmfs7g/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-cookies-6.04-6ry3g43avudslc67nmarfuwdcfvco7uo/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-message-6.13-abohrnrxuisfhk3t72seiv574bc3v4mc/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-io-html-1.001-oyqd2erifynoeupo66hjcaycihsicoqq/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-lwp-mediatypes-6.02-bdp3xi7cv2fdcauoihwrard42kepoxia/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-try-tiny-0.28-zvtluoeno7jqe67p2qzwwubchfxa4kz4/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-uri-1.72-o37xz4ilf4rnmda2svvtotxm4xeam2ui/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-test-needs-0.002005-tnrswihcn4zxwfksomggqzarz33rnl7y/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-daemon-6.01-rko6s7gedbap2xxkdpus4zp74zt3dvfw/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-module-build-tiny-0.039-mmndrhwxlan4msx5ad2c5ohhwjpiu33m/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-extutils-config-0.008-ftjvuhon5zjjsgap5ortxwd7xbkshsph/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-extutils-helpers-0.026-stpvmn3lj5ue2qkblyys53jkuza6cl57/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-extutils-installpaths-0.012-uwbd4w3fdwy5mmsqfjvryiw2eyopbz2z/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-module-build-0.4224-t3cop5s3douirz5q55bvwbkhfadwd4el/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-http-negotiate-6.01-ommofkbrsj7u6ainlq5eeksb5nlcgymc/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-net-http-6.17-omo2m5srfju7nxpljyxzjashh2unpzvg/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-www-robotrules-6.02-uf5nlvfqshw7rbvtpnzcrmfaj5djiun2/lib/perl5:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/perl-data-dumper-2.173-hxgfazm472z25crph4yyvurdrwr37563/lib/perl5; export PERL5LIB
PKG_CONFIG_PATH=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/cairo-1.16.0-2jluafsrkf5neehysrkso5tau5ihxcf6/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/renderproto-0.11.1-emcx3vuhvblg44pvs3c4sor5ihdfyc5w/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxdmcp-1.1.2-6izlm3rd6hazdewvnultwwbhakjdksbm/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libx11-1.6.7-xiug4qf74mreb4awmuiyhdx3jxwxlc3x/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fontconfig-2.12.3-zct5jz7bzilfao2tbhm56rxt2ezrw4of/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxcb-1.13-4it4r7ddtsqomtfb2b73i64ekghovboo/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xextproto-7.3.0-mu4d2pxsnfovkjf5xiz2jifvmr7mvhfl/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/zlib-1.2.11-6jyao6lrmcwyq7czbm6rkotf2kotdkms/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxau-1.0.8-j5wav4lxjwarzlydjjgxka3kbkqm6gyf/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/atk-2.30.0-donciffdv5sjdflnt2o3o3rac7fgkelc/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pango-1.41.0-7fm7ua5nhdamvfcwwpcdeafk35zdabqu/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pcre-8.42-grb7dd2yxp6dik2wpk3gcxxviccn66i6/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpthread-stubs-0.4-6y5jawqhzywitj5qau4zytfqrhe322nj/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fixesproto-5.0-s2y5coudhip4s5tmbtnm45izlqx5v6li/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/readline-8.0-7s3tknmsgd2ogp5i2fxanncjwwlz5imd/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/dbus-1.12.8-qz3lq5alzkjcu7bo2txav35upfarsg7w/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xz-5.2.4-fczgkodctns3rrkjxydtw7utm3hgtszb/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libffi-3.2.1-5wasckgclanmn27zgih4gqfdg72gnsts/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxext-1.3.3-as6foiyy32m47pzhscqpq6alvs373c33/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/expat-2.2.9-mnfcw7ow22ukhji4ypq43ty4qu454f33/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/mesa-18.3.6-m2njhpfhxpxn3nth5sws5z7ys65dzvbh/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pixman-0.38.0-altkkt5wlktit32npwgogpi7qlfap4qi/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xproto-7.0.31-caalnyvvme5wfiac6mkraqu2jggxug4o/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpng-1.6.37-znjevoldm5l2cmwr5w6mzwqcp6mdiieq/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/openssl-1.1.1d-q2wzstunrtm256ctqhqzplruwi7msedn/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-core-2.28.0-74hsipz5ctek25xcecrjk5puevmli7wl/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxrender-0.9.10-cdsuuaxbchxcgupekrozvxs3h2ncgnhz/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/kbproto-1.0.7-bbyqmd4rg7nd2bck6tur6lahnw6hvojd/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxml2-2.9.9-rsomm2rw7otzkjbqja46yfr3i4dk32bk/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/shared-mime-info-1.9-4c7mld6ugi4fxaqiwz5e6o747vw7e25v/share/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/harfbuzz-2.3.1-3f2tn3mmfa2c5w5z2syhengrhv3qbh7f/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdk-pixbuf-2.38.0-fer36fxn3xqf53yrohvbh6vxqylvkjfg/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libepoxy-1.4.3-3jhwzitxd6kem47je74xtojbzw2egvcb/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/font-util-1.3.2-wmlz4nwnnj7jhxbrg6age6ziw5nwrs5v/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxft-2.3.2-wozy34ikvfbplhmk6mfwcapt5jtllt75/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/freetype-2.10.1-3nyzzr7hrfmrtvjziucbhf7ls2m5mocq/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/inputproto-2.3.2-nundvjesicbpkcrosjvwhggrzbt2gh3j/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/sqlite-3.30.1-sbgek46asjt6gx475p3o264aoalkiiez/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gtkplus-3.20.10-z7424szywk3cog6dzgfhxo5xdwug3jno/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxi-1.7.6-zhwscljazuzdxd7d4h4brynj22wwbpx6/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libtiff-4.0.10-56m4wnmsgy6speokfawaw7ezn3pjnpha/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gobject-introspection-1.56.1-dxqvlfxwwkvvlxex2nzgnl3u6u6p42zb/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/icu4c-64.1-lmyzw76ojwniyhgq3ztwmgniihnbjtfd/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libedit-3.1-20170329-ceqgzwvgiar3mswya7iv3ayjokncfkzi/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/python-3.7.4-k4keiszfsytvqkmrn44g3dbglkn3n7iq/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libbsd-0.9.1-nzolvhj75tmbzgtsycmnmispon2a5lne/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-atk-2.26.2-wwqrhwzbcelwyfwcsifykyrbufgj2t4g/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxfixes-5.0.2-ld3isqoniw52emmjc4dgej3fsfrkiv4n/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/glib-2.56.3-ncbtvgcc33233flrixxmuyhex54nqafu/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/ncurses-6.1-giqjoryaltcr2gj227fdhapqzozqbxue/lib/pkgconfig:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libjpeg-turbo-2.0.3-qu75rijvkj3bvptcyx3v5wmyqjm6a72u/lib/pkgconfig; export PKG_CONFIG_PATH
PULSE_SCRIPT=/etc/xrdp/pulse/default.pa; export PULSE_SCRIPT
#PWD=/home/vsoni/tools/spack/bin; export PWD
PYTHONHOME=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/python-3.7.4-k4keiszfsytvqkmrn44g3dbglkn3n7iq; export PYTHONHOME
PYTHONPATH=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/meson-0.49.1-5u5phgd74iiao54lfeh5urqzslrmkwuq/lib/python3.7/site-packages:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/py-setuptools-41.4.0-2aghkfra4453czdfzsgst4vvwxdh4env/lib/python3.7/site-packages:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/llvm-9.0.0-q5vpxksi4qdfaonkvqwtejh24wtf3eqw/lib/python3.7/site-packages:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/py-mako-1.0.4-ya2lobqkf3gy2omtpvocv26ng3hg4vbu/lib/python3.7/site-packages:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/py-markupsafe-1.1.1-t6n3evqpej7mtg7gj46vdhbjkgd2kci2/lib/python3.7/site-packages; export PYTHONPATH
QT4_IM_MODULE=xim; export QT4_IM_MODULE
QT_ACCESSIBILITY=1; export QT_ACCESSIBILITY
QT_IM_MODULE=ibus; export QT_IM_MODULE
SESSION_MANAGER=local/vsoni-Virtual-Machine:@/tmp/.ICE-unix/3241,unix/vsoni-Virtual-Machine:/tmp/.ICE-unix/3241; export SESSION_MANAGER
SHELL=/bin/bash; export SHELL
SHLVL=1; export SHLVL
SPACK_CC=/home/vsoni/tools/gcc-9.2.0/bin/gcc; export SPACK_CC
SPACK_CC_RPATH_ARG=-Wl,-rpath,; export SPACK_CC_RPATH_ARG
SPACK_COMPILER_IMPLICIT_RPATHS=/home/vsoni/tools/gcc-9.2.0/lib/gcc/x86_64-pc-linux-gnu/9.2.0:/home/vsoni/tools/gcc-9.2.0/lib64; export SPACK_COMPILER_IMPLICIT_RPATHS
SPACK_COMPILER_SPEC=gcc@9.2.0; export SPACK_COMPILER_SPEC
SPACK_CXX=/home/vsoni/tools/gcc-9.2.0/bin/g++; export SPACK_CXX
SPACK_CXX_RPATH_ARG=-Wl,-rpath,; export SPACK_CXX_RPATH_ARG
SPACK_DEBUG_LOG_DIR=/home/vsoni/tools/spack/bin; export SPACK_DEBUG_LOG_DIR
SPACK_DEBUG_LOG_ID=wxpropgrid-4pyki3l; export SPACK_DEBUG_LOG_ID
SPACK_DTAGS_TO_ADD=--disable-new-dtags; export SPACK_DTAGS_TO_ADD
SPACK_DTAGS_TO_STRIP=--enable-new-dtags; export SPACK_DTAGS_TO_STRIP
SPACK_ENV_PATH=/home/vsoni/tools/spack/lib/spack/env:/home/vsoni/tools/spack/lib/spack/env/case-insensitive:/home/vsoni/tools/spack/lib/spack/env/gcc; export SPACK_ENV_PATH
SPACK_F77=/home/vsoni/tools/gcc-9.2.0/bin/gfortran; export SPACK_F77
SPACK_F77_RPATH_ARG=-Wl,-rpath,; export SPACK_F77_RPATH_ARG
SPACK_FC=/home/vsoni/tools/gcc-9.2.0/bin/gfortran; export SPACK_FC
SPACK_FC_RPATH_ARG=-Wl,-rpath,; export SPACK_FC_RPATH_ARG
SPACK_INCLUDE_DIRS=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/bzip2-1.0.8-bl277ctefakb6ff4kxdxkxxpgtc7zpdk/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libjpeg-turbo-2.0.3-qu75rijvkj3bvptcyx3v5wmyqjm6a72u/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/ncurses-6.1-giqjoryaltcr2gj227fdhapqzozqbxue/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/glib-2.56.3-ncbtvgcc33233flrixxmuyhex54nqafu/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxfixes-5.0.2-ld3isqoniw52emmjc4dgej3fsfrkiv4n/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-atk-2.26.2-wwqrhwzbcelwyfwcsifykyrbufgj2t4g/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libbsd-0.9.1-nzolvhj75tmbzgtsycmnmispon2a5lne/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/python-3.7.4-k4keiszfsytvqkmrn44g3dbglkn3n7iq/include/python3.7m:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libedit-3.1-20170329-ceqgzwvgiar3mswya7iv3ayjokncfkzi/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/icu4c-64.1-lmyzw76ojwniyhgq3ztwmgniihnbjtfd/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gobject-introspection-1.56.1-dxqvlfxwwkvvlxex2nzgnl3u6u6p42zb/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libtiff-4.0.10-56m4wnmsgy6speokfawaw7ezn3pjnpha/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxi-1.7.6-zhwscljazuzdxd7d4h4brynj22wwbpx6/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gtkplus-3.20.10-z7424szywk3cog6dzgfhxo5xdwug3jno/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/sqlite-3.30.1-sbgek46asjt6gx475p3o264aoalkiiez/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/inputproto-2.3.2-nundvjesicbpkcrosjvwhggrzbt2gh3j/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/binutils-2.32-ehnjwaojurg4n6vbbgsdfp7aho7qwvvo/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/freetype-2.10.1-3nyzzr7hrfmrtvjziucbhf7ls2m5mocq/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxft-2.3.2-wozy34ikvfbplhmk6mfwcapt5jtllt75/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libepoxy-1.4.3-3jhwzitxd6kem47je74xtojbzw2egvcb/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdbm-1.18.1-o3gn2zefwtl2qopdw3hpznlzxbm6ufvd/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdk-pixbuf-2.38.0-fer36fxn3xqf53yrohvbh6vxqylvkjfg/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/harfbuzz-2.3.1-3f2tn3mmfa2c5w5z2syhengrhv3qbh7f/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxml2-2.9.9-rsomm2rw7otzkjbqja46yfr3i4dk32bk/include/libxml2:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/kbproto-1.0.7-bbyqmd4rg7nd2bck6tur6lahnw6hvojd/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxrender-0.9.10-cdsuuaxbchxcgupekrozvxs3h2ncgnhz/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-core-2.28.0-74hsipz5ctek25xcecrjk5puevmli7wl/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/openssl-1.1.1d-q2wzstunrtm256ctqhqzplruwi7msedn/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpng-1.6.37-znjevoldm5l2cmwr5w6mzwqcp6mdiieq/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xproto-7.0.31-caalnyvvme5wfiac6mkraqu2jggxug4o/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pixman-0.38.0-altkkt5wlktit32npwgogpi7qlfap4qi/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/mesa-18.3.6-m2njhpfhxpxn3nth5sws5z7ys65dzvbh/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxdmcp-1.1.2-6izlm3rd6hazdewvnultwwbhakjdksbm/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxext-1.3.3-as6foiyy32m47pzhscqpq6alvs373c33/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libffi-3.2.1-5wasckgclanmn27zgih4gqfdg72gnsts/lib/libffi-3.2.1/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxwidgets-3.1.0-zartvb5bcg5mddj4n7smzindfkcbg7mq/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xz-5.2.4-fczgkodctns3rrkjxydtw7utm3hgtszb/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/dbus-1.12.8-qz3lq5alzkjcu7bo2txav35upfarsg7w/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/readline-8.0-7s3tknmsgd2ogp5i2fxanncjwwlz5imd/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fixesproto-5.0-s2y5coudhip4s5tmbtnm45izlqx5v6li/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pcre-8.42-grb7dd2yxp6dik2wpk3gcxxviccn66i6/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pango-1.41.0-7fm7ua5nhdamvfcwwpcdeafk35zdabqu/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/atk-2.30.0-donciffdv5sjdflnt2o3o3rac7fgkelc/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxau-1.0.8-j5wav4lxjwarzlydjjgxka3kbkqm6gyf/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libiconv-1.16-qhwzzrmbtm6myg523nyh6yb42q4tuam3/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/llvm-9.0.0-q5vpxksi4qdfaonkvqwtejh24wtf3eqw/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/zlib-1.2.11-6jyao6lrmcwyq7czbm6rkotf2kotdkms/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gettext-0.20.1-xno722wxyggfz6ako7u6dnc2vikmmv4e/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xextproto-7.3.0-mu4d2pxsnfovkjf5xiz2jifvmr7mvhfl/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxcb-1.13-4it4r7ddtsqomtfb2b73i64ekghovboo/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fontconfig-2.12.3-zct5jz7bzilfao2tbhm56rxt2ezrw4of/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libx11-1.6.7-xiug4qf74mreb4awmuiyhdx3jxwxlc3x/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/expat-2.2.9-mnfcw7ow22ukhji4ypq43ty4qu454f33/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/renderproto-0.11.1-emcx3vuhvblg44pvs3c4sor5ihdfyc5w/include:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/cairo-1.16.0-2jluafsrkf5neehysrkso5tau5ihxcf6/include; export SPACK_INCLUDE_DIRS
SPACK_LINKER_ARG=-Wl,; export SPACK_LINKER_ARG
SPACK_LINK_DIRS=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/bzip2-1.0.8-bl277ctefakb6ff4kxdxkxxpgtc7zpdk/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libjpeg-turbo-2.0.3-qu75rijvkj3bvptcyx3v5wmyqjm6a72u/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/ncurses-6.1-giqjoryaltcr2gj227fdhapqzozqbxue/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/glib-2.56.3-ncbtvgcc33233flrixxmuyhex54nqafu/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxfixes-5.0.2-ld3isqoniw52emmjc4dgej3fsfrkiv4n/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-atk-2.26.2-wwqrhwzbcelwyfwcsifykyrbufgj2t4g/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libbsd-0.9.1-nzolvhj75tmbzgtsycmnmispon2a5lne/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/python-3.7.4-k4keiszfsytvqkmrn44g3dbglkn3n7iq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libedit-3.1-20170329-ceqgzwvgiar3mswya7iv3ayjokncfkzi/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/icu4c-64.1-lmyzw76ojwniyhgq3ztwmgniihnbjtfd/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gobject-introspection-1.56.1-dxqvlfxwwkvvlxex2nzgnl3u6u6p42zb/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libtiff-4.0.10-56m4wnmsgy6speokfawaw7ezn3pjnpha/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxi-1.7.6-zhwscljazuzdxd7d4h4brynj22wwbpx6/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gtkplus-3.20.10-z7424szywk3cog6dzgfhxo5xdwug3jno/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/sqlite-3.30.1-sbgek46asjt6gx475p3o264aoalkiiez/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/inputproto-2.3.2-nundvjesicbpkcrosjvwhggrzbt2gh3j/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/binutils-2.32-ehnjwaojurg4n6vbbgsdfp7aho7qwvvo/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/freetype-2.10.1-3nyzzr7hrfmrtvjziucbhf7ls2m5mocq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxft-2.3.2-wozy34ikvfbplhmk6mfwcapt5jtllt75/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/font-util-1.3.2-wmlz4nwnnj7jhxbrg6age6ziw5nwrs5v/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libepoxy-1.4.3-3jhwzitxd6kem47je74xtojbzw2egvcb/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdbm-1.18.1-o3gn2zefwtl2qopdw3hpznlzxbm6ufvd/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdk-pixbuf-2.38.0-fer36fxn3xqf53yrohvbh6vxqylvkjfg/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/harfbuzz-2.3.1-3f2tn3mmfa2c5w5z2syhengrhv3qbh7f/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxml2-2.9.9-rsomm2rw7otzkjbqja46yfr3i4dk32bk/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/kbproto-1.0.7-bbyqmd4rg7nd2bck6tur6lahnw6hvojd/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxrender-0.9.10-cdsuuaxbchxcgupekrozvxs3h2ncgnhz/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-core-2.28.0-74hsipz5ctek25xcecrjk5puevmli7wl/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/openssl-1.1.1d-q2wzstunrtm256ctqhqzplruwi7msedn/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpng-1.6.37-znjevoldm5l2cmwr5w6mzwqcp6mdiieq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xproto-7.0.31-caalnyvvme5wfiac6mkraqu2jggxug4o/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pixman-0.38.0-altkkt5wlktit32npwgogpi7qlfap4qi/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/mesa-18.3.6-m2njhpfhxpxn3nth5sws5z7ys65dzvbh/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxdmcp-1.1.2-6izlm3rd6hazdewvnultwwbhakjdksbm/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxext-1.3.3-as6foiyy32m47pzhscqpq6alvs373c33/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libffi-3.2.1-5wasckgclanmn27zgih4gqfdg72gnsts/lib64:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libffi-3.2.1-5wasckgclanmn27zgih4gqfdg72gnsts/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxwidgets-3.1.0-zartvb5bcg5mddj4n7smzindfkcbg7mq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xz-5.2.4-fczgkodctns3rrkjxydtw7utm3hgtszb/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/dbus-1.12.8-qz3lq5alzkjcu7bo2txav35upfarsg7w/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/readline-8.0-7s3tknmsgd2ogp5i2fxanncjwwlz5imd/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fixesproto-5.0-s2y5coudhip4s5tmbtnm45izlqx5v6li/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpthread-stubs-0.4-6y5jawqhzywitj5qau4zytfqrhe322nj/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pcre-8.42-grb7dd2yxp6dik2wpk3gcxxviccn66i6/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pango-1.41.0-7fm7ua5nhdamvfcwwpcdeafk35zdabqu/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/atk-2.30.0-donciffdv5sjdflnt2o3o3rac7fgkelc/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxau-1.0.8-j5wav4lxjwarzlydjjgxka3kbkqm6gyf/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libiconv-1.16-qhwzzrmbtm6myg523nyh6yb42q4tuam3/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/llvm-9.0.0-q5vpxksi4qdfaonkvqwtejh24wtf3eqw/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/zlib-1.2.11-6jyao6lrmcwyq7czbm6rkotf2kotdkms/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gettext-0.20.1-xno722wxyggfz6ako7u6dnc2vikmmv4e/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xextproto-7.3.0-mu4d2pxsnfovkjf5xiz2jifvmr7mvhfl/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxcb-1.13-4it4r7ddtsqomtfb2b73i64ekghovboo/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fontconfig-2.12.3-zct5jz7bzilfao2tbhm56rxt2ezrw4of/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libx11-1.6.7-xiug4qf74mreb4awmuiyhdx3jxwxlc3x/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/meson-0.49.1-5u5phgd74iiao54lfeh5urqzslrmkwuq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/expat-2.2.9-mnfcw7ow22ukhji4ypq43ty4qu454f33/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/renderproto-0.11.1-emcx3vuhvblg44pvs3c4sor5ihdfyc5w/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/cairo-1.16.0-2jluafsrkf5neehysrkso5tau5ihxcf6/lib; export SPACK_LINK_DIRS
SPACK_ROOT=/home/vsoni/tools/spack; export SPACK_ROOT
SPACK_RPATH_DIRS=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxpropgrid-1.4.15-4pyki3ldvmeelfu52ntq22baqy2nte5o/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxpropgrid-1.4.15-4pyki3ldvmeelfu52ntq22baqy2nte5o/lib64:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/bzip2-1.0.8-bl277ctefakb6ff4kxdxkxxpgtc7zpdk/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libjpeg-turbo-2.0.3-qu75rijvkj3bvptcyx3v5wmyqjm6a72u/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/ncurses-6.1-giqjoryaltcr2gj227fdhapqzozqbxue/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/glib-2.56.3-ncbtvgcc33233flrixxmuyhex54nqafu/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxfixes-5.0.2-ld3isqoniw52emmjc4dgej3fsfrkiv4n/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-atk-2.26.2-wwqrhwzbcelwyfwcsifykyrbufgj2t4g/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libbsd-0.9.1-nzolvhj75tmbzgtsycmnmispon2a5lne/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/python-3.7.4-k4keiszfsytvqkmrn44g3dbglkn3n7iq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libedit-3.1-20170329-ceqgzwvgiar3mswya7iv3ayjokncfkzi/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/icu4c-64.1-lmyzw76ojwniyhgq3ztwmgniihnbjtfd/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gobject-introspection-1.56.1-dxqvlfxwwkvvlxex2nzgnl3u6u6p42zb/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libtiff-4.0.10-56m4wnmsgy6speokfawaw7ezn3pjnpha/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxi-1.7.6-zhwscljazuzdxd7d4h4brynj22wwbpx6/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gtkplus-3.20.10-z7424szywk3cog6dzgfhxo5xdwug3jno/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/sqlite-3.30.1-sbgek46asjt6gx475p3o264aoalkiiez/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/inputproto-2.3.2-nundvjesicbpkcrosjvwhggrzbt2gh3j/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/binutils-2.32-ehnjwaojurg4n6vbbgsdfp7aho7qwvvo/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/freetype-2.10.1-3nyzzr7hrfmrtvjziucbhf7ls2m5mocq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxft-2.3.2-wozy34ikvfbplhmk6mfwcapt5jtllt75/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/font-util-1.3.2-wmlz4nwnnj7jhxbrg6age6ziw5nwrs5v/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libepoxy-1.4.3-3jhwzitxd6kem47je74xtojbzw2egvcb/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdbm-1.18.1-o3gn2zefwtl2qopdw3hpznlzxbm6ufvd/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdk-pixbuf-2.38.0-fer36fxn3xqf53yrohvbh6vxqylvkjfg/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/harfbuzz-2.3.1-3f2tn3mmfa2c5w5z2syhengrhv3qbh7f/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxml2-2.9.9-rsomm2rw7otzkjbqja46yfr3i4dk32bk/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/kbproto-1.0.7-bbyqmd4rg7nd2bck6tur6lahnw6hvojd/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxrender-0.9.10-cdsuuaxbchxcgupekrozvxs3h2ncgnhz/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/at-spi2-core-2.28.0-74hsipz5ctek25xcecrjk5puevmli7wl/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/openssl-1.1.1d-q2wzstunrtm256ctqhqzplruwi7msedn/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpng-1.6.37-znjevoldm5l2cmwr5w6mzwqcp6mdiieq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xproto-7.0.31-caalnyvvme5wfiac6mkraqu2jggxug4o/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pixman-0.38.0-altkkt5wlktit32npwgogpi7qlfap4qi/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/mesa-18.3.6-m2njhpfhxpxn3nth5sws5z7ys65dzvbh/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxdmcp-1.1.2-6izlm3rd6hazdewvnultwwbhakjdksbm/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxext-1.3.3-as6foiyy32m47pzhscqpq6alvs373c33/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libffi-3.2.1-5wasckgclanmn27zgih4gqfdg72gnsts/lib64:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libffi-3.2.1-5wasckgclanmn27zgih4gqfdg72gnsts/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/wxwidgets-3.1.0-zartvb5bcg5mddj4n7smzindfkcbg7mq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xz-5.2.4-fczgkodctns3rrkjxydtw7utm3hgtszb/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/dbus-1.12.8-qz3lq5alzkjcu7bo2txav35upfarsg7w/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/readline-8.0-7s3tknmsgd2ogp5i2fxanncjwwlz5imd/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fixesproto-5.0-s2y5coudhip4s5tmbtnm45izlqx5v6li/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libpthread-stubs-0.4-6y5jawqhzywitj5qau4zytfqrhe322nj/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pcre-8.42-grb7dd2yxp6dik2wpk3gcxxviccn66i6/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pango-1.41.0-7fm7ua5nhdamvfcwwpcdeafk35zdabqu/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/atk-2.30.0-donciffdv5sjdflnt2o3o3rac7fgkelc/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxau-1.0.8-j5wav4lxjwarzlydjjgxka3kbkqm6gyf/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libiconv-1.16-qhwzzrmbtm6myg523nyh6yb42q4tuam3/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/llvm-9.0.0-q5vpxksi4qdfaonkvqwtejh24wtf3eqw/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/zlib-1.2.11-6jyao6lrmcwyq7czbm6rkotf2kotdkms/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gettext-0.20.1-xno722wxyggfz6ako7u6dnc2vikmmv4e/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/xextproto-7.3.0-mu4d2pxsnfovkjf5xiz2jifvmr7mvhfl/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libxcb-1.13-4it4r7ddtsqomtfb2b73i64ekghovboo/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/fontconfig-2.12.3-zct5jz7bzilfao2tbhm56rxt2ezrw4of/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libx11-1.6.7-xiug4qf74mreb4awmuiyhdx3jxwxlc3x/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/meson-0.49.1-5u5phgd74iiao54lfeh5urqzslrmkwuq/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/expat-2.2.9-mnfcw7ow22ukhji4ypq43ty4qu454f33/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/renderproto-0.11.1-emcx3vuhvblg44pvs3c4sor5ihdfyc5w/lib:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/cairo-1.16.0-2jluafsrkf5neehysrkso5tau5ihxcf6/lib; export SPACK_RPATH_DIRS
SPACK_SHORT_SPEC='wxpropgrid@1.4.15%gcc@9.2.0 arch=linux-ubuntu18.04-skylake/4pyki3l'; export SPACK_SHORT_SPEC
SPACK_SYSTEM_DIRS=/bin:/usr/bin:/usr/local/bin:/bin64:/usr/bin64:/usr/local/bin64:/include:/usr/include:/usr/local/include:/lib:/usr/lib:/usr/local/lib:/lib64:/usr/lib64:/usr/local/lib64:/:/usr:/usr/local; export SPACK_SYSTEM_DIRS
SPACK_TARGET_ARGS='-march=skylake -mtune=skylake'; export SPACK_TARGET_ARGS
SSH_AGENT_PID=3378; export SSH_AGENT_PID
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh; export SSH_AUTH_SOCK
TERM=xterm-256color; export TERM
TEXTDOMAIN=im-config; export TEXTDOMAIN
TEXTDOMAINDIR=/usr/share/locale/; export TEXTDOMAINDIR
UID=1000; export UID
USER=vsoni; export USER
VTE_VERSION=5202; export VTE_VERSION
XDG_CURRENT_DESKTOP=ubuntu:GNOME; export XDG_CURRENT_DESKTOP
XDG_DATA_DIRS=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gtkplus-3.20.10-z7424szywk3cog6dzgfhxo5xdwug3jno/share:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/pango-1.41.0-7fm7ua5nhdamvfcwwpcdeafk35zdabqu/share:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gdk-pixbuf-2.38.0-fer36fxn3xqf53yrohvbh6vxqylvkjfg/share:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/shared-mime-info-1.9-4c7mld6ugi4fxaqiwz5e6o747vw7e25v/share:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/atk-2.30.0-donciffdv5sjdflnt2o3o3rac7fgkelc/share:/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/gobject-introspection-1.56.1-dxqvlfxwwkvvlxex2nzgnl3u6u6p42zb/share:/usr/share/gnome:/usr/local/share:/usr/share:/var/lib/snapd/desktop; export XDG_DATA_DIRS
XDG_MENU_PREFIX=gnome-; export XDG_MENU_PREFIX
XDG_RUNTIME_DIR=/run/user/1000; export XDG_RUNTIME_DIR
XDG_SESSION_ID=c3; export XDG_SESSION_ID
XLOCALEDIR=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/libx11-1.6.7-xiug4qf74mreb4awmuiyhdx3jxwxlc3x/share/X11/locale; export XLOCALEDIR
XML_CATALOG_FILES=/home/vsoni/tools/spack/opt/spack/linux-ubuntu18.04-skylake/gcc-9.2.0/docbook-xsl-1.79.2-6stus3wwk4z365bg6z2irpns4lo7ijxo/catalog.xml; export XML_CATALOG_FILES
XMODIFIERS=@im=ibus; export XMODIFIERS
XRDP_SESSION=1; export XRDP_SESSION
XRDP_SOCKET_PATH=/var/run/xrdp/sockdir; export XRDP_SOCKET_PATH
_=/home/vsoni/tools/spack/bin/spack; export _
```
</details>

I am having the same issue with the configuration of wxpropgrid. It looks like wxpropgrid is expecting either gtk1 or gtk2, while wxwidgets is using gtk3. So, I tried:

`spack install wxpropgrid ^gtkplus@2.24.32`

and this got me past the configuration phase:

```
checking for wx-config... /home/angelv/spack/opt/spack/linux-archrolling-ivybridge/gcc-8.3.0/wxwidgets-3.1.0-rdbumw6ppqpjbtoubpcfb263q4mjo54d/bin/wx-config
checking for wxWidgets version >= 2.6.1 (--unicode=yes)... yes (version 3.1.0)
checking for wxWidgets static library... no
checking if wxWidgets was built in DEBUG mode... no
checking if wxWidgets was built in STATIC mode... no
checking which wxWidgets toolkit was selected... gtk2
checking for ranlib... ranlib
```

but then I get redefinition errors when compiling:

```
In file included from /home/angelv/spack/opt/spack/linux-archrolling-ivybridge/gcc-8.3.0/wxwidgets-3.1.0-rdbumw6ppqpjbtoubpcfb263q4mjo54d/include/wx-3.1/wx/event.h:25,
                 from /home/angelv/spack/opt/spack/linux-archrolling-ivybridge/gcc-8.3.0/wxwidgets-3.1.0-rdbumw6ppqpjbtoubpcfb263q4mjo54d/include/wx-3.1/wx/app.h:19,
                 from ./src/odcombo.cpp:29:
./include/wx/propgrid/propgrid.h:3927:43: error: redefinition of 'class wxArrayDouble'
     WX_DECLARE_OBJARRAY_WITH_DECL(double, wxArrayDouble, class WXDLLIMPEXP_PG);
                                           ^~~~~~~~~~~~~
/home/angelv/spack/opt/spack/linux-archrolling-ivybridge/gcc-8.3.0/wxwidgets-3.1.0-rdbumw6ppqpjbtoubpcfb263q4mjo54d/include/wx-3.1/wx/dynarray.h:576:10: note: in definition of macro '_WX_DECLARE_OBJARRAY'
 classexp name : protected base                                           \
          ^~~~

```

Any ideas?

--- a/var/spack/repos/builtin/packages/paraver/package.py
+++ b/var/spack/repos/builtin/packages/paraver/package.py
@@ -7,25 +7,18 @@ from spack import *
 import os
 
 
-class Paraver(AutotoolsPackage):
+class Paraver(Package):
     """"A very powerful performance visualization and analysis tool
         based on traces that can be used to analyse any information that
         is expressed on its input trace format.  Traces for parallel MPI,
-        OpenMP and other programs can be genereated with Extrae."""
+        OpenMP and other programs can be genereated with Extrae.
+
+        Note: This package is deprecated in favor of wxparaver"""
     homepage = "https://tools.bsc.es/paraver"
     url = "https://ftp.tools.bsc.es/wxparaver/wxparaver-4.6.3-src.tar.bz2"
 
     # NOTE: Paraver provides only latest version for download.
     #       Don't keep/add older versions.
-    version('4.9.2',     sha256='83289584040bcedf8cab1b2ae3545191c8bdef0e11ab62b06e54cbf111f2127a', deprecated=True)
-    version('4.9.1',     sha256='e89fdf563d1fc73ed0018cf0e25b458b6617ec33325ed3fdbf06397c556f3a8e', deprecated=True)
-    version('4.9.0',     sha256='1f9964d7987032d01a354327845bf53ae369be5d8acf7d4e85bec81699a6ddf6', deprecated=True)
-    version('4.8.2',     sha256='0d22ec362e4798d7ed21b15b607859e9bda9579e3d5b23af3812c1e32ccc192d', deprecated=True)
-    version('4.8.1',     sha256='d03b04254bd3015d61374b95aeda6888f593be67286c5268849623baa2ae6e2e', deprecated=True)
-    version('4.8.0',     sha256='780af8fff7cb40d1325260fb9f79210f6676f07357bc9b95b1b838862f2d1e5b', deprecated=True)
-    version('4.7.2',     sha256='90107797d6af6fc3ebd9505445bb518d673edecbe5d08d1b7af01695d53241ae', deprecated=True)
-    version('4.7.1',     sha256='8cbec0c5e0f8a849820f6682cbb0920ea234bb7f20d1483e38ea5d0b0ee045cd', deprecated=True)
-    version('4.7.0',     sha256='81e02bcc1853455b13435172a4336ba85ba05020887d322c9678c97def03d76f', deprecated=True)
     version('4.6.3',     sha256='ac6025eec5419e1060967eab71dfd123e585be5b5f3ac3241085895dbeca255a', deprecated=True)
     version('4.6.2',     sha256='74b85bf9e6570001d372b376b58643526e349b1d2f1e7633ca38bb0800ecf929', deprecated=True)
 
@@ -35,33 +28,6 @@ class Paraver(AutotoolsPackage):
     depends_on('libxml2')
     depends_on('zlib')
 
-    def configure_args(self):
-        spec = self.spec
-        args = []
-
-        if spec.satisfies('@4.6.3'):
-            args.append('--with-paraver=%s' % self.prefix)
-            args.append('--with-ptools-common-files=%s' % self.prefix)
-        args.append('--with-boost=%s' % spec['boost'].prefix)
-        args.append('--with-wx-config=%s/wx-config' % spec['wxwidgets'].prefix.bin)
-        if spec['wxwidgets'].satisfies('@:2'):
-            args.append('--with-wxpropgrid=%s' % spec['wxpropgrid'].prefix)
-
-        return args
-
-    @when('@4.6.2')  # do not have global build system
-    def autoreconf(self, spec, prefix):
-        pass
-
-    @when('@4.6.2')  # do not have global build system
-    def configure(self, spec, prefix):
-        pass
-
-    # use make install directly as expected by Paraver
-    def build(self, spec, prefix):
-        pass
-
-    @when('@4.6.2')  # do not have global build system
     def install(self, spec, prefix):
         os.chdir("ptools_common_files")
         configure("--prefix=%s" % prefix)

--- a/var/spack/repos/builtin/packages/paraver/package.py
+++ b/var/spack/repos/builtin/packages/paraver/package.py
@@ -7,24 +7,61 @@ from spack import *
 import os
 
 
-class Paraver(Package):
+class Paraver(AutotoolsPackage):
     """"A very powerful performance visualization and analysis tool
         based on traces that can be used to analyse any information that
         is expressed on its input trace format.  Traces for parallel MPI,
         OpenMP and other programs can be genereated with Extrae."""
     homepage = "https://tools.bsc.es/paraver"
-    url = "https://ftp.tools.bsc.es/paraver/wxparaver-4.6.3-src.tar.bz2"
+    url = "https://ftp.tools.bsc.es/wxparaver/wxparaver-4.6.3-src.tar.bz2"
 
     # NOTE: Paraver provides only latest version for download.
     #       Don't keep/add older versions.
-    version('4.6.3', sha256='ac6025eec5419e1060967eab71dfd123e585be5b5f3ac3241085895dbeca255a')
-    version('4.6.2', sha256='74b85bf9e6570001d372b376b58643526e349b1d2f1e7633ca38bb0800ecf929')
+    version('4.9.2',     sha256='83289584040bcedf8cab1b2ae3545191c8bdef0e11ab62b06e54cbf111f2127a')
+    version('4.9.1',     sha256='e89fdf563d1fc73ed0018cf0e25b458b6617ec33325ed3fdbf06397c556f3a8e')
+    version('4.9.0',     sha256='1f9964d7987032d01a354327845bf53ae369be5d8acf7d4e85bec81699a6ddf6')
+    version('4.8.2',     sha256='0d22ec362e4798d7ed21b15b607859e9bda9579e3d5b23af3812c1e32ccc192d')
+    version('4.8.1',     sha256='d03b04254bd3015d61374b95aeda6888f593be67286c5268849623baa2ae6e2e')
+    version('4.8.0',     sha256='780af8fff7cb40d1325260fb9f79210f6676f07357bc9b95b1b838862f2d1e5b')
+    version('4.7.2',     sha256='90107797d6af6fc3ebd9505445bb518d673edecbe5d08d1b7af01695d53241ae')
+    version('4.7.1',     sha256='8cbec0c5e0f8a849820f6682cbb0920ea234bb7f20d1483e38ea5d0b0ee045cd')
+    version('4.7.0',     sha256='81e02bcc1853455b13435172a4336ba85ba05020887d322c9678c97def03d76f')
+    version('4.6.3',     sha256='ac6025eec5419e1060967eab71dfd123e585be5b5f3ac3241085895dbeca255a')
+    version('4.6.2',     sha256='74b85bf9e6570001d372b376b58643526e349b1d2f1e7633ca38bb0800ecf929', deprecated=True)
 
-    depends_on("boost")
-    # depends_on("extrae")
-    depends_on("wxwidgets")
-    depends_on("wxpropgrid")
+    depends_on('boost@1.36: +serialization')
+    depends_on('wxwidgets@2.8:')  # NOTE: using external for this one is usually simpler
+    depends_on('wxpropgrid@1.4:')
+    depends_on('libxml2')
+    depends_on('zlib')
 
+    def configure_args(self):
+        spec = self.spec
+        args = []
+
+        if spec.satisfies('@4.6.3'):
+            args.append('--with-paraver=%s' % self.prefix)
+            args.append('--with-ptools-common-files=%s' % self.prefix)
+        args.append('--with-boost=%s' % spec['boost'].prefix)
+        args.append('--with-wx-config=%s/wx-config' % spec['wxwidgets'].prefix.bin)
+        if spec['wxwidgets'].satisfies('@:2'):
+            args.append('--with-wxpropgrid=%s' % spec['wxpropgrid'].prefix)
+
+        return args
+
+    @when('@4.6.2')  # do not have global build system
+    def autoreconf(self, spec, prefix):
+        pass
+
+    @when('@4.6.2')  # do not have global build system
+    def configure(self, spec, prefix):
+        pass
+
+    # use make install directly as expected by Paraver
+    def build(self, spec, prefix):
+        pass
+
+    @when('@4.6.2')  # do not have global build system
     def install(self, spec, prefix):
         os.chdir("ptools_common_files")
         configure("--prefix=%s" % prefix)

--- a/var/spack/repos/builtin/packages/wxparaver/package.py
+++ b/var/spack/repos/builtin/packages/wxparaver/package.py
@@ -4,7 +4,6 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
-import os
 
 
 class Wxparaver(AutotoolsPackage):
@@ -15,8 +14,6 @@ class Wxparaver(AutotoolsPackage):
     homepage = "https://tools.bsc.es/paraver"
     url = "https://ftp.tools.bsc.es/wxparaver/wxparaver-4.6.3-src.tar.bz2"
 
-    # NOTE: Paraver provides only latest version for download.
-    #       Don't keep/add older versions.
     version('4.9.2',     sha256='83289584040bcedf8cab1b2ae3545191c8bdef0e11ab62b06e54cbf111f2127a')
     version('4.9.1',     sha256='e89fdf563d1fc73ed0018cf0e25b458b6617ec33325ed3fdbf06397c556f3a8e')
     version('4.9.0',     sha256='1f9964d7987032d01a354327845bf53ae369be5d8acf7d4e85bec81699a6ddf6')
@@ -26,8 +23,6 @@ class Wxparaver(AutotoolsPackage):
     version('4.7.2',     sha256='90107797d6af6fc3ebd9505445bb518d673edecbe5d08d1b7af01695d53241ae')
     version('4.7.1',     sha256='8cbec0c5e0f8a849820f6682cbb0920ea234bb7f20d1483e38ea5d0b0ee045cd')
     version('4.7.0',     sha256='81e02bcc1853455b13435172a4336ba85ba05020887d322c9678c97def03d76f')
-    version('4.6.3',     sha256='ac6025eec5419e1060967eab71dfd123e585be5b5f3ac3241085895dbeca255a')
-    version('4.6.2',     sha256='74b85bf9e6570001d372b376b58643526e349b1d2f1e7633ca38bb0800ecf929', deprecated=True)
 
     depends_on('boost@1.36: +serialization')
     depends_on('wxwidgets@2.8:')  # NOTE: using external for this one is usually simpler
@@ -39,55 +34,13 @@ class Wxparaver(AutotoolsPackage):
         spec = self.spec
         args = []
 
-        if spec.satisfies('@4.6.3'):
-            args.append('--with-paraver=%s' % self.prefix)
-            args.append('--with-ptools-common-files=%s' % self.prefix)
         args.append('--with-boost=%s' % spec['boost'].prefix)
         args.append('--with-wx-config=%s/wx-config' % spec['wxwidgets'].prefix.bin)
-        if spec['wxwidgets'].satisfies('@:2'):
+        if spec['wxwidgets'].satisfies('@:2.999'):
             args.append('--with-wxpropgrid=%s' % spec['wxpropgrid'].prefix)
 
         return args
 
-    @when('@4.6.2')  # do not have global build system
-    def autoreconf(self, spec, prefix):
-        pass
-
-    @when('@4.6.2')  # do not have global build system
-    def configure(self, spec, prefix):
-        pass
-
-    # use make install directly as expected by Paraver
+    # use make install directly as expected by Paraver (See README)
     def build(self, spec, prefix):
         pass
-
-    @when('@4.6.2')  # do not have global build system
-    def install(self, spec, prefix):
-        os.chdir("ptools_common_files")
-        configure("--prefix=%s" % prefix)
-        make()
-        make("install")
-
-        os.chdir("../paraver-kernel")
-        # "--with-extrae=%s" % spec['extrae'].prefix,
-        configure("--prefix=%s" % prefix,
-                  "--with-ptools-common-files=%s" % prefix,
-                  "--with-boost=%s" % spec['boost'].prefix,
-                  "--with-boost-serialization=boost_serialization")
-        make()
-        make("install")
-
-        os.chdir("../paraver-toolset")
-        configure("--prefix=%s" % prefix)
-        make()
-        make("install")
-
-        os.chdir("../wxparaver")
-        # "--with-extrae=%s" % spec['extrae'].prefix,
-        configure("--prefix=%s" % prefix,
-                  "--with-paraver=%s" % prefix,
-                  "--with-boost=%s" % spec['boost'].prefix,
-                  "--with-boost-serialization=boost_serialization",
-                  "--with-wxdir=%s" % spec['wxwidgets'].prefix.bin)
-        make()
-        make("install")

--- a/var/spack/repos/builtin/packages/wxparaver/package.py
+++ b/var/spack/repos/builtin/packages/wxparaver/package.py
@@ -12,7 +12,7 @@ class Wxparaver(AutotoolsPackage):
         is expressed on its input trace format.  Traces for parallel MPI,
         OpenMP and other programs can be genereated with Extrae."""
     homepage = "https://tools.bsc.es/paraver"
-    url = "https://ftp.tools.bsc.es/wxparaver/wxparaver-4.6.3-src.tar.bz2"
+    url = "https://ftp.tools.bsc.es/wxparaver/wxparaver-4.9.2-src.tar.bz2"
 
     version('4.9.2',     sha256='83289584040bcedf8cab1b2ae3545191c8bdef0e11ab62b06e54cbf111f2127a')
     version('4.9.1',     sha256='e89fdf563d1fc73ed0018cf0e25b458b6617ec33325ed3fdbf06397c556f3a8e')

--- a/var/spack/repos/builtin/packages/wxparaver/package.py
+++ b/var/spack/repos/builtin/packages/wxparaver/package.py
@@ -7,7 +7,7 @@ from spack import *
 import os
 
 
-class Paraver(AutotoolsPackage):
+class Wxparaver(AutotoolsPackage):
     """"A very powerful performance visualization and analysis tool
         based on traces that can be used to analyse any information that
         is expressed on its input trace format.  Traces for parallel MPI,
@@ -17,16 +17,16 @@ class Paraver(AutotoolsPackage):
 
     # NOTE: Paraver provides only latest version for download.
     #       Don't keep/add older versions.
-    version('4.9.2',     sha256='83289584040bcedf8cab1b2ae3545191c8bdef0e11ab62b06e54cbf111f2127a', deprecated=True)
-    version('4.9.1',     sha256='e89fdf563d1fc73ed0018cf0e25b458b6617ec33325ed3fdbf06397c556f3a8e', deprecated=True)
-    version('4.9.0',     sha256='1f9964d7987032d01a354327845bf53ae369be5d8acf7d4e85bec81699a6ddf6', deprecated=True)
-    version('4.8.2',     sha256='0d22ec362e4798d7ed21b15b607859e9bda9579e3d5b23af3812c1e32ccc192d', deprecated=True)
-    version('4.8.1',     sha256='d03b04254bd3015d61374b95aeda6888f593be67286c5268849623baa2ae6e2e', deprecated=True)
-    version('4.8.0',     sha256='780af8fff7cb40d1325260fb9f79210f6676f07357bc9b95b1b838862f2d1e5b', deprecated=True)
-    version('4.7.2',     sha256='90107797d6af6fc3ebd9505445bb518d673edecbe5d08d1b7af01695d53241ae', deprecated=True)
-    version('4.7.1',     sha256='8cbec0c5e0f8a849820f6682cbb0920ea234bb7f20d1483e38ea5d0b0ee045cd', deprecated=True)
-    version('4.7.0',     sha256='81e02bcc1853455b13435172a4336ba85ba05020887d322c9678c97def03d76f', deprecated=True)
-    version('4.6.3',     sha256='ac6025eec5419e1060967eab71dfd123e585be5b5f3ac3241085895dbeca255a', deprecated=True)
+    version('4.9.2',     sha256='83289584040bcedf8cab1b2ae3545191c8bdef0e11ab62b06e54cbf111f2127a')
+    version('4.9.1',     sha256='e89fdf563d1fc73ed0018cf0e25b458b6617ec33325ed3fdbf06397c556f3a8e')
+    version('4.9.0',     sha256='1f9964d7987032d01a354327845bf53ae369be5d8acf7d4e85bec81699a6ddf6')
+    version('4.8.2',     sha256='0d22ec362e4798d7ed21b15b607859e9bda9579e3d5b23af3812c1e32ccc192d')
+    version('4.8.1',     sha256='d03b04254bd3015d61374b95aeda6888f593be67286c5268849623baa2ae6e2e')
+    version('4.8.0',     sha256='780af8fff7cb40d1325260fb9f79210f6676f07357bc9b95b1b838862f2d1e5b')
+    version('4.7.2',     sha256='90107797d6af6fc3ebd9505445bb518d673edecbe5d08d1b7af01695d53241ae')
+    version('4.7.1',     sha256='8cbec0c5e0f8a849820f6682cbb0920ea234bb7f20d1483e38ea5d0b0ee045cd')
+    version('4.7.0',     sha256='81e02bcc1853455b13435172a4336ba85ba05020887d322c9678c97def03d76f')
+    version('4.6.3',     sha256='ac6025eec5419e1060967eab71dfd123e585be5b5f3ac3241085895dbeca255a')
     version('4.6.2',     sha256='74b85bf9e6570001d372b376b58643526e349b1d2f1e7633ca38bb0800ecf929', deprecated=True)
 
     depends_on('boost@1.36: +serialization')

